### PR TITLE
add playground settings option

### PR DESCRIPTION
**Pull request by MomenNano**
fixed [272](https://github.com/mcollina/fastify-gql/issues/272)

this RP is WIP, its missing documentation and testing (I'm not sure how to test it)

but I want to make sure I got the settings option name right.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ const plugin = fp(async function (app, opts) {
       errorHandler: opts.errorHandler,
       errorFormatter: opts.errorFormatter,
       ide: optsIde,
+      ideSettings: opts.playgroundOpts,
```
Comment (mcollina) [inline index.js:202]: Could you please add the option to the README? I would call it `playgroundSettings` as it's not available for graphiql.

**Review by mcollina — COMMENTED** [review]
You can add a new test where the plugin is configured with `playgroundSettings` and then you use the snapshot test to check this (as it is right now for `undefined`).

File: index.js (hunk 1 above)
Comment (MomenNano) [reply index.js]: okay

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by MomenNano** [comment]
first I want to thank you very much for helping me with this PR <3

I did all the things you mentioned. I think it's ready to be reviewed.

**Pull request merged**